modules/getent/getent-group.c

```diff
@@ -20,6 +20,8 @@
  * COPYING for details.
  */
 
+#include "compat.h"
```

Comment:
These files are included into the `tfgetent.c`, which is not the best idea imho.
But if you don't plan to resolve this incleption I think this `compat.h` should be in the `tfgentent.c`. Because it would mix the two concept.
Of course the preferred fix would be to make these files properly independent compilation chunk.